src/stack_pr/shell_commands.py

```diff
@@ -32,15 +44,16 @@ def run_shell_command(
     if "shell" in kwargs:
         raise ValueError("shell support has been removed")
     _ = subprocess.list2cmdline(cmd)
-    kwargs.update({"check": check})
     if quiet:
         kwargs.update({"stdout": subprocess.DEVNULL, "stderr": subprocess.DEVNULL})
-    if SHOW_COMMANDS:
-        print(f"Running: {cmd}")
-    return subprocess.run(list(map(str, cmd)), **kwargs)
+    logger.debug("Running: %s", cmd)
```

Comment:
Could you please double check that adding `--verbose` does result in showing commands being run with this change?

Reply:
yeah. nothing printed.